cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/PdfReportBuilder.java

```diff
@@ -22,9 +23,13 @@
 
     PdfReportBuilder addElement(ReportElement element);
 
-    PdfReportBuilder withHeaderOnAllPages(String left, String middle, String right);
+    PdfReportBuilder withHeaderOnAllPages(String left, String middle, String right, boolean excludeOnAllPages);
 
-    PdfReportBuilder withFooterOnAllPages(String left, String middle, String right);
+    PdfReportBuilder withHeaderOnAllPages(ReportElement headerElement, boolean excludeOnFirstPage);
+
+    PdfReportBuilder withFooterOnAllPages(String left, String middle, String right, boolean excludeOnAllPages);
```

Comment:
here you call the new boolean parameter `excludeOnAllPages` in the impl `excludeOnFirstPage` - which one is it now?